starwars/src/components/StarCard.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Card, CardText } from "reactstrap";
+
+const StarCard = props => {
+  console.log(props.character);
+  console.log(props.howTall);
+  console.log(props.weight);
+  console.log(props.birth_year);
+
+  return (
```

Comment:
awesome work! you can also shorten this by destructuring, like this 
`const { howTall, weight, birth_year } = props.character`
and then you don't have to reference props each time, you can just write `{weight}` etc